scripts/pylocks_generator.py

```
@@ -0,0 +1,411 @@
+# PRE-FLIGHT CHECK
+# =============================================================================
+
+
+def check_uv() -> None:
+    """Verify the uv wrapper exists and meets the minimum version requirement."""
+    if not UV.is_file() or not os.access(UV, os.X_OK):
+        error(f"Expected uv wrapper at '{UV}' but it is missing or not executable.")
+        raise SystemExit(1)
+
+    try:
+        result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)